.travis.yml

```diff
@@ -1,24 +1,27 @@
 # Travis CI configuration file for running tests
 language: python
+sudo: false
 python:
   - "2.7"
 env:
   - BOTO_CONFIG=/dev/null
+branches:
+  only:
+    - master
 before_install:
-  - pip install pep8
-  - pep8 --verbose
+  - export BOTO_CONFIG=/dev/null
```

Comment:
we have this in `env:`, which is better?

Reply:
ours is 8 months old, edX's 6 months old

Reply:
Ours seems more correct, since it's prescribed by Travis-CI;
lets use it.